tests/test.bats

```diff
@@ -5,7 +5,7 @@ exec_sql() {
     ../build/bin/mysql -uroot <$1
 }
 
-setup() {
+setup_file() {
```

Comment:
[TODO]
mysqld frontend may cache the return values of `SELECT` queries.
We should restart MySQL server for each testcase.

Reply:
reverted.